src/css/tabs/auxiliary.less

```diff
@@ -1,5 +1,5 @@
 .tab-auxiliary {
-	height: 100%;
+	min-height: 100%;
```

Comment:
Seems to work without:

```suggestion
```

Reply:
Have you tried with only one or two modes and the rest hidden?

Reply:
Only tested scrolling. Hide switch does not seem to work with `yarn dev`.

Reply:
It works in my case, at least if I have one more configured. If there is none it does not work.